docs/guide/migrating-to-version-9.md

```diff
@@ -570,6 +570,21 @@ Removed as of `v9`. Use `KBreadcumbs` instead.
 
 ### KToaster
 
+#### Attributes
+
+#### Structure
+
+* `toaster-container-outer` class has been replaced with `k-toaster`
+* `toaster-item` class has been replaced with `toast`
```

Comment:
seems like we're switching between "toaster" and "toast" - can we pick one? 

I would think "toaster" since that's the name of the component?

Reply:
Selectors, the constants and types, etc.

Reply:
Updated - only left the `Toast` interface name unchanged because it was called that before and I think it makes sense (interface of a **single Toast**)